src/util/GeoSparqlHelpers.h

```diff
@@ -0,0 +1,107 @@
+// Copyright 2021, University of Freiburg,
+// Author: Hannah Bast <bast@cs.uni-freiburg.de>
+// Chair of Algorithms and Data Structures
+
+#ifndef QLEVER_GEOSPARQLHELPERS_H
+#define QLEVER_GEOSPARQLHELPERS_H
+
+#include <string>
+
+// TODO: How can I declare the lambdas in the .h file and implement them in the
+// .cpp file? The best I managed so far is define them with inline in the .f
+// file. If I define them without inline in the .cpp file, I get a multiple
+// definition error.
```

Comment:
With lambdas this simply doesn't work, and the current Expression framework basically requires a lambda.
We could change this, but it would be quite some code, so the inline lambdas are currently the best solution.
But when restructuring the Expression module we can discuss this.

Reply:
It was hard work to get it to compile with the implementations in the .cpp files (partly due to my unfamiliarity with the cmake setup), but it seems to work now.